test/internals/reply.test.js

```diff
@@ -477,6 +477,35 @@ test('stream with content type should not send application/octet-stream', t => {
   })
 })
 
+test('stream without content type should not send application/octet-stream', t => {
+  t.plan(4)
+
+  const fastify = require('../..')()
+  const fs = require('fs')
+  const path = require('path')
```

Comment:
why make these requires inside the test?

Reply:
For this test I just followed the setup of the test above so I have no justification for this setup. I'll clean it all up to be simpler